enterprise/backend/src/metabase_enterprise/serialization/v2/load.clj

```diff
@@ -92,7 +93,10 @@
                              (throw (ex-info (format "Failed to read file for %s" (serdes/log-path-str path))
                                              (path-error-data ::not-found expanding path)
                                              e))))
+                ;; if no :entity_id specified we must guarantee a new entity is created
+                require-new-entity (nil? (:entity_id ingested ::not-found))
```

Comment:
I think it would be worth calling out another bit of context here: any truly exported entity should have `:entity_id` set, since we backfill before export. So if it's blank in the incoming file then it's been removed on purpose. Likewise we did a backfill before import, so the appdb cannot have any blank `entity_id`s in it either.

Reply:
I've been a bit more explicit in the comment.  Hope its an improvement.